expect/_matchers.ts

```diff
@@ -375,11 +375,15 @@ export function toContain(
 
   if (context.isNot) {
     if (doesContain) {
-      throw new AssertionError("The value contains the expected item");
+      throw new AssertionError(
+        `The value "${context.value}" contains the expected item "${expected}"`,
```

Comment:
Can you use `format` function instead? `format` handles the case when the value has `"` in it.